app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError)
```

Comment:
```suggestion
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
    )
```

Reply:
Thanks for the comments, I already implemented these changes :)